{"repo_id": "Automattic_mongoose_14587", "bugs": [{"description": "Possible crash in error-handling logic. The code uses `err instanceof mongoose.mongo.MongoWriteConcernError` (e.g. in test/util.js line 23) without first confirming that `mongoose.mongo.MongoWriteConcernError` is defined and callable. If the MongoDB driver version being used does not define this class, `instanceof undefined` will throw a `TypeError`, causing the helper to fail while processing an error.\n\nSafer pattern:\n```\nif (typeof mongoose.mongo.MongoWriteConcernError === 'function' &&\n    err instanceof mongoose.mongo.MongoWriteConcernError &&\n    /operation was interrupted/.test(err.message)) {\n  ...\n}\n```", "file": "test/util.js", "line": 23}], "updated_files": {}}